docs/guide/advanced-configuration.md

```diff
@@ -0,0 +1,89 @@
+## Advanced Configuration
+
+### Docker
+
+If the `docker` configuration option is set to `true`, Docker will be used to start Webots, otherwise Webots will be started directly on the metal of the server.
+If the simulation project contains a `Dockerfile` file at the root level, this file will be used to start the Webots instance inside the corresponding Docker container.
+Otherwise, a `Dockerfile` will be created based on the standard Docker image of Webots.
+The version of Webots for the Docker image is automatically computed from the header line of the simulation world file.
+For example if the world file starts with the following line:
+
+```
+#VRML_SIM R2022a utf8
+```
+
+The simulation server will create a `Dockerfile` starting with:
+```
+FROM docker image cyberbotics/webots:R2022a-ubuntu20.04
+```
+
+Running Webots inside a Docker container is a very little overhead, but guarantees that the simulation server remain secure, regardless of the running simulations.
+
+If you don't want to use Docker, you should ensure that the list of `allowedRepositories` provided in the configuration file doesn't contain any malware, otherwise, you are putting your simulation server at risk.
```

Comment:
```suggestion
If you don't want to use Docker, you should ensure that the list of `allowedRepositories` provided in the configuration file doesn't contain any links to malicious repositories, otherwise, you are putting your simulation server at risk.
```